lit_nlp/examples/models/tydi.py

```diff
@@ -0,0 +1,176 @@
+"""LIT wrappers for T5, supporting both HuggingFace and SavedModel formats."""
+import re
+from typing import List
+
+import attr
+from lit_nlp.api import model as lit_model
+from lit_nlp.api import types as lit_types
+from lit_nlp.examples.models import model_utils
+from lit_nlp.lib import utils
+
+import tensorflow as tf
+# tensorflow_text is required for T5 SavedModel
+import tensorflow_text  # pylint: disable=unused-import
+import transformers
+from transformers import BertTokenizer
+from transformers import FlaxBertForQuestionAnswering
+
+from rouge_score import rouge_scorer
+
+JsonDict = lit_types.JsonDict
```

Comment:
```suggestion
import transformers

from rouge_score import rouge_scorer

BertTokenizer = transformers.BertTokenizer
FlaxBertForQuestionAnswering = transformers.FlaxBertForQuestionAnswering
JsonDict = lit_types.JsonDict
```

This is how we get around [the import restrictions](https://google.github.io/styleguide/pyguide.html#22-imports) in Google's Python Style Guide. Since we can't import the classes directly, and writing `transformers.{ClassName}` is less readable than we might want, we create local variables with the same names as the classes we want and assign them the value from the relevant modules.